samples/shadow/v2/README.md

```diff
@@ -0,0 +1,127 @@
+# Shadow
```

Comment:
This readme isn't adapted to a new sample. From what I see, the description is not true anymore, the running commands are wrong. Also, is QOS section still relevant for new clients?